libraries/AP_GPS/AP_GPS_GSOF.cpp

```diff
@@ -71,17 +71,29 @@ AP_GPS_GSOF::AP_GPS_GSOF(AP_GPS &_gps, AP_GPS::GPS_State &_state,
 
     const uint32_t now = AP_HAL::millis();
     gsofmsg_time = now + 110;
+
+    const auto raw_data = gps._raw_data.get();
```

Comment:
I'm not sure this logic is going to work. You can't just check individual bytes until they match what you expect, because they are interleaved with the stream data. Do you have any other approaches to try?

Reply:
> For retrieving a parameter from the param server, you can do it like so: ``const auto com_port = gps._com_port[state.instance].get()``

I was actually a bit confused how to go about this, since you mentioned in the ticket, I should be using the .get() method, but I was not able figure out exactly where do they get ``_raw_data`` value in ``AP_GPS_UBLOX.cpp`` or ``AP_GPS_SBF.cpp`` ( I couldn't find a get() method for the parameter, nor any other initialization except the one at ``AP_GPS.cpp`` ), the value is clearly using to trigger disarm cases.
Could you guide me here on how I should be going about it?